plugins/multinet.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multinet
+spec:
+  version: v0.2.1
+  homepage: https://github.com/k8snetworkplumbingwg/kubectl-multinet
+  shortDescription: "Shows pods' network-status annotation of NPWG multi-net-spec"
+  description: |
+    Shows pods' network-status annotation, defined in NPWG multi-net-spec.
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
```

Comment:
most plugins also provide a darwin build for macOS (many Kubernetes users use mac).

Reply:
Will do but it might be a bit later, not now.